docs/manual/guides/tinymce-configuration.de.md

```diff
@@ -0,0 +1,219 @@
+---
+title: "TinyMCE-Editor Konfiguration"
+description: "Die TinyMCE-Editor Konfiguration"
+url: "anleitungen/tinymce-konfiguration"
+weight: 40
+---
+
+
+Zur Bearbeitung der meisten Inhalte wird der [TinyMCE-Editor](https://www.tiny.cloud/) herangezogen. 
+Der Editor ist für Contao bereits vorkonfiguriert. Individuelle Änderungen sind optional. 
+
+
+## Das Template be_tinyMCE.html5
+
+Deine individuelle Konfiguration erfolgt über das Template `be_tinyMCE.html5`. Hier findest du die von Contao gesetzte
+Konfiguration vor und kannst die Angaben updatesicher anpassen.
+
+Erstelle über den Navigationsbereich »Layout« unter »Templates« ein neues [Template](../../theme-manager/templates-verwalten). 
+Wähle über `Original-Template` das Template `be_tinyMCE.html5` aus und gebe als `Zielverzeichnis` das Hauptverzeichnis an. 
+
+{{% notice note %}}
+Das Template **muss** im Hauptverzeichnis (`templates/be_tinyMCE.html5`) abgelegt werden und du solltest die Datei
+in keinem Fall umbenennen. Alle Zeilen innerhalb von `<script>...</script>` bis auf die letzte Zeile müssen 
```

Comment:
> und du solltest die Datei in keinem Fall umbenennen

Das ist so nicht richtig. Du kannst die Datei nennen wie du willst - denn du kannst verschiedene RTE Konfigurationen auswählen. Im DCA kannst du diese RTE Konfiguration dann definieren. Siehe https://docs.contao.org/dev/reference/dca/fields/#evaluation

Reply:
Habe das geändert - Das mit den DCA ...? Soll ich das aufnehmen bzw. hast du dazu einen Vorschlag? Bin mir dazu nicht sicher.

Reply:
Ja, das sollte erwähnt werden. Also einfach ein Beispiel, wie man eine andere TinyMCE Konfiguration für z. B. `tl_news` machen kann.

Reply:
Thanks - need help on this

Reply:
Was ist nicht klar? Vielleicht hilft folgender [Link](https://de.contaowiki.org/TinyMCE).